lib/periodicChecks.js

```diff
@@ -0,0 +1,49 @@
+const assignUnassignedReviewers = async (pullRequest, context) => {
+  const assignees = pullRequest.assignees.map(user => user.login);
+  const reviewers = pullRequest.requested_reviewers.map(user => user.login);
+
+  // If the PR author is assigned, then don't assign other reviewers.
+  if (assignees.includes(pullRequest.user.login)) {
+    return;
+  }
+
+  let botComment = 'Hey ';
```

Comment:
Can we use a string template here (or in any such places), as it will make it easier for us to understand the whole message? (not sure, @ankita240796 any thoughts?)

Reply:
Using a template string here will cause linting errors since there is no javascript interpolation.

Reply:
I see, thanks for check. Can we generate this string in one place (like [here](https://github.com/oppia/oppiabot/blob/master/lib/checkPullRequestLabels.js#L43)) that will make it more readable, maybe?

Reply:
But this would not work for arrays since we could be pinging multiple reviewers at the same time.